src/App.js

```diff
@@ -2,12 +2,17 @@ import React, { Component } from 'react';
 import './App.css';
 import Video from './Video/Video'
 import Header from './Component/Header/Header'
+import KikkPro from './Component/KikkPro/KikkPro';
+import Logo from './Logo/Logo';
```

Comment:
folder name should be Image instead of logo

Reply:
Also, change the Logo to Image folder